docs/library/pyb.UART.rst

```diff
@@ -111,11 +111,6 @@ Methods
 
        Returns the number of bytes waiting (may be 0).
 
-    .. method:: UART.writechar(char)
-
-      Write a single character on the bus.  ``char`` is an integer to write.
-      Return value: ``None``. See note below if CTS flow control is used.
-
```

Comment:
Reordering the description for this method is a separate thing, nothing to do with timeout, so please put it in a separate commit (but same PR if you like).